### DISCUSSION TITLE
spatie media library not working with bower sync in port 3000

### DISCUSSION THREAD

**Discussion by ivnlabs** [spatie/laravel-medialibrary#2549] [Q&A]
I am trying to upload images with the plugin but I cant 
I use bower sync with webpack but I can upload when my local host is port8000 not 3000 the one that refresh alone, how to make this work?

**Comment by Anticom** [comment]
https://webpack.js.org/configuration/dev-server/#devserverproxy

**Comment by ivnlabs** [comment]
Hi there, I love to work with mix.browserSync because my browser can update
the page every time I change something on the code of the dev app, BUT I am
having a hard time trying to use the http://localhost:3000/ with spatie,

Everything works great but when I want to upload images with spatie media
library the upload bar still updating the information forever , this
happens when I am in http://localhost:3000/

If I want to upload the images I have to go to http://localhost:8000/ and
here the page works great but do not auto-refresh on the moment I change
the code because the mix browser sync got this code to work

if (!useChat) {

   mix.browserSync({
        proxy: 'http://localhost:3000',
        notify: false
    });

hope make sense, What I want to achive is auto load page on code change AND
also if I want to upload the images with this spatie plugin it can be done
in http://localhost:3000/ this way I don't have to write every time the
route because I am in dev right now and its time consuming doing that
change form 3000 to 8000 on and on

thanks a lot for taking the time for reading my post

On Tue, Jul 27, 2021 at 6:53 AM Timo Mühlbach ***@***.***>
wrote:

> https://webpack.js.org/configuration/dev-server/#devserverproxy
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/spatie/laravel-medialibrary/issues/2512#issuecomment-887448331>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AP7YXOXRJAIKQK5Y7DBGKTTTZ2M4NANCNFSM5AI7EKRA>
> .
>


-- 

Espero haber aclarado todas sus dudas en esta cotización, cualquier
inquietud no dude en escribirnos de nuevo.

Cordialmente

InverNorth Team
***@***.***
